server/middleware/setUpWebSecurity.ts

```diff
@@ -24,10 +24,17 @@ export default function setUpWebSecurity(): Router {
             "'self'",
             (req, res: Response) => `'nonce-${res.locals.cspNonce}'`,
             'code.jquery.com',
+            'www.googletagmanager.com',
+            'www.google-analytics.com',
```

Comment:
need `https://` on these ones?

Reply:
No i don't think so - had it blocking them before this change locally, and allowed them after.